lametro/static/js/autocomplete.js

```diff
@@ -9,59 +8,33 @@ var SmartLogic = {
 
     if ( (tokenNeeded || tokenExpired) ) {
       return $.get(
-        '/ses-token/'
+        '/smartlogic/token/'
```

Comment:
The changes in this file are mostly related to factoring out the complicated response processing (it is now done in Python).